### PR TITLE
Feature/poss indicator

### PR DESCRIPTION
**Pull request by klein1992**

### CODE REVIEW

File: Assets/Prefabs/ProjectorAnimation.anim [hunk 1]
```diff
@@ -0,0 +1,116 @@
+%YAML 1.1
```
Comment (Jazy5552) [inline Assets/Prefabs/ProjectorAnimation.anim]: We have an animations folder for a reason

File: Assets/Scripts/Entity/Player/Possession.cs [hunk 2]
```diff
@@ -2,49 +2,95 @@
 using System.Collections.Generic;
 using UnityEngine;
 
+
```
Comment (Jazy5552) [inline Assets/Scripts/Entity/Player/Possession.cs]: What's with all these extra new lines? Trying to increase the lines of code you have commited? hu?

File: Assets/Scripts/Entity/Player/Possession.cs [hunk 3]
```diff
@@ -112,7 +160,7 @@ private void NotifyOnPossessionControllers(GameObject enemy)
             }
         }
         return nearestEnemy;
-
+        
     }
-
+
```
Comment (Jazy5552) [inline Assets/Scripts/Entity/Player/Possession.cs]: Porque all these whitespaces? They actually trigger my IDE
image

**Review by Jazy5552 — DISMISSED** [review]
Don't forget to either add or let someone know about the floor having to be on the UI layer for the projector to work.

File: Assets/Scripts/Entity/Player/Possession.cs (hunk 2 above)
Comment (klein1992) [reply Assets/Scripts/Entity/Player/Possession.cs]: lol guilty as charged. Jk not intentional

Comment (klein1992) [reply Assets/Scripts/Entity/Player/Possession.cs]: What had happened was I was needed a namespace but ended up not needing it...

File: Assets/Prefabs/ProjectorAnimation.anim (hunk 1 above)
Comment (klein1992) [reply Assets/Prefabs/ProjectorAnimation.anim]: I need to clean my room and transfer that organization in my life lol

File: Assets/Scripts/Entity/Player/Possession.cs (hunk 3 above)
Comment (klein1992) [reply Assets/Scripts/Entity/Player/Possession.cs]: lol no idea why that happened

**Review by AleksandrHovhannisyan — CHANGES_REQUESTED** [review]
Dude, you're killing me. Please change the Room prefab's floor, not Jazy's sandbox floor... Otherwise we're gonna have to do another PR to get this to actually work in Level1.

**Comment by klein1992** [comment]
I added 2 commits last night...one of them changed the actual room prefab floor....

**Comment by AleksandrHovhannisyan** [comment]
It doesn't work on Level1.

**Review by AleksandrHovhannisyan — APPROVED** [review]

**Pull request merged**